examples/function_todomvc/src/components/input.rs

```diff
@@ -0,0 +1,33 @@
+use web_sys::HtmlInputElement;
+use yew::events::KeyboardEvent;
+use yew::{function_component, html, Callback, Properties, TargetCast};
+
+#[derive(PartialEq, Properties, Clone)]
+pub struct InputProps {
+    pub add: Callback<String>,
+}
+
+#[function_component(Input)]
```

Comment:
This component name should be more descriptive because it is for a very particular purpose and I don't want it to confuse people between `input` and `Input`.